### PR TITLE
Make workspaces tasks default…

### PR DESCRIPTION
**Pull request by vdemeester**
- Rename tasks with pipeline resources with `-pr`
- Rename tasks with workspace dropping `-workspace` suffix
- Updates README(s), task names, …

Related to https://issues.redhat.com/browse/SRVKP-824

/cc @chmouel @piyush-garg @savitaashture @pradeepitm12 @siamaksade @sthaha @nikhil-thomas @khrm 

Signed-off-by: Vincent Demeester <vincent@sbr.pm>

### CODE REVIEW

**Comment by chmouel** [comment]
Do we need to keep the -pr if they are not "favoured" ? isnt that going to be kinda confusing ?

**Comment by vdemeester** [comment]
> Do we need to keep the -pr if they are not "favoured" ? isnt that going to be kinda confusing ?

the `-pr` wasn't there before, and instead we had `-workspace` on the workspaces ones. I think we can keep the `-pr` for existing user that are using / want to use PipelineResource. @siamaksade wdyt ?

**Comment by siamaksade** [comment]
@vdemeester agreed. We can remove the -pr ones in the next one or two releases

File: task/s2i-dotnet-1-pr/0.1/s2i-dotnet-1.yaml [hunk 1]
```diff
@@ -1,16 +1,16 @@
 apiVersion: tekton.dev/v1beta1
```
Comment (piyush-garg) [inline task/s2i-dotnet-1-pr/0.1/s2i-dotnet-1.yaml:1]: File name should be s2i-dotnet-1-pr.yaml

File: task/s2i-dotnet-1/0.1/README.md [hunk 2]
```diff
@@ -62,34 +53,70 @@ oc adm policy add-scc-to-user privileged -z pipeline
 oc adm policy add-role-to-user edit -z pipeline
 ```
 
-## Creating the taskrun
+## Using a a `Pipeline` with `git-clone`
```
Comment (piyush-garg) [inline task/s2i-dotnet-1/0.1/README.md:56]: nit: remove one `a`

File: task/s2i-dotnet-1-pr/0.1/s2i-dotnet-1.yaml (hunk 1 above)
Comment (piyush-garg) [reply task/s2i-dotnet-1-pr/0.1/s2i-dotnet-1.yaml:1]: This applies for all the  *-pr tasks

File: task/s2i-dotnet-1/0.1/s2i-dotnet-1.yaml [hunk 3]
```diff
@@ -6,11 +6,11 @@ metadata:
     app.kubernetes.io/version: "0.1"
   annotations:
     tekton.dev/pipelines.minVersion: "0.11.3"
-    tekton.dev/tags: s2i, dotnet
+    tekton.dev/tags: s2i, dotnet, workspace
     tekton.dev/displayName: "s2i dotnet 1"
 spec:
   description: >-
-    s2i-dotnet-1 task fetches a Git repository and builds and
+    s2i-dotnet-1-workspace task clones a Git repository and builds and
```
Comment (piyush-garg) [inline task/s2i-dotnet-1/0.1/s2i-dotnet-1.yaml:13]: need to remove  `-workspace`

**Review by nikhil-thomas — APPROVED** [review]
/lgtm

removing `-worspace` tag from the workspace tasks in this catalog makes sense.

However, when we add them to the operator, we might want to specify the `-workspace` suffix and `version`. Otherwise it might break user pipelines using our ClusterTasks (which use `pipelineResource` by default`).

**Pull request merged**